### PR TITLE
Adding tests for the module

### PR DESCRIPTION
**Pull request by reficul31**
I would have preferred to have the test side by side of the index file but due to the increased simplicity of the babel configuration and its nature to pull all the js files into build, I have put the test in its own directory.

### CODE REVIEW

**Review by Treora — COMMENTED** [review]
Putting it in a separate folder seems fine too; there are not many source files in this repo anyway so it should be easy enough to find it. :)

I don't think you actually need the jest-fetch-mock here, since the code does not use fetch at all. You only use it to create a Response object in the test, but for that you could just do what jest-fetch-mock would now do internally: call `const response = new Response(imageBlob)`. See [docs](https://developer.mozilla.org/en-US/docs/Web/API/Response/Response).

You probably have to `import 'whatwg-fetch'` to be able to use `Response` in Nodejs.

File: package.json [hunk 1]
```diff
@@ -23,6 +24,9 @@
     "babel-preset-stage-1": "^6.24.1"
   },
   "dependencies": {
+    "blob-util": "^1.2.1",
+    "jest": "^20.0.4",
+    "jest-fetch-mock": "^1.2.0",
```
Comment (Treora) [inline package.json]: can be removed now (along with its deps in yarn.lock, if any)

File: __test__/index.test.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+/* eslint-env jest */
+
+import responseToDataURI from '../src/index'
+import { dataURLToBlob } from 'blob-util'
+import 'whatwg-fetch'
```
Comment (Treora) [inline __test__/index.test.js]: detail: I always sort the imports to first have external libs, then an empty line and then local imports. Might be a habit stemming from Python, but I found it a good habit.

File: __test__/index.test.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+const imageDataUrl = 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAAAAAA6fptVAAAACklEQVR4nGNiAAAABgADNjd8qAAAAABJRU5ErkJggg=='
+
+describe('responseToDataURI tests', () => {
+	let imageBlob
```
Comment (Treora) [inline __test__/index.test.js]: indentation

File: __test__/index.test.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+/* eslint-env jest */
+
+import responseToDataURI from '../src/index'
```
Comment (Treora) [inline __test__/index.test.js]: I renamed data URI to URL everywhere; please update all occurrences of 'uri' in this file too.

**Review by Treora — COMMENTED** [review]
Just some details.

**Comment by reficul31** [comment]
@Treora almost ready to merge?

**Comment by Treora** [comment]
Looks good to me! Let's push that button.

**Pull request merged**